chef/cookbooks/tempest/templates/default/tempest.conf.erb

```diff
@@ -411,7 +411,7 @@ api_v3_extensions = all
 
 # Does the test environment use block devices for live migration
 # (boolean value)
-#block_migration_for_live_migration = false
+block_migration_for_live_migration = true
```

Comment:
Shouldn't that depend on the option to use shared storage for the instances data?